ibm/mas_devops/roles/suite_app_rollback/README.md

```diff
@@ -0,0 +1,77 @@
+suite_app_rollback
+===============================================================================
+This role is to roll back Maximo Application Suite Applications to an earlier version. Currently, this is designed for Manage Application only. Rollback is possible only in 8.7 and later. From 8.7 onwards, every version comes with a set of supported versions to which the Application can be rolled back. For example, you can roll back Manage Application from 8.7.x to 8.7.0. 
+This role will rollback the version for an an installed MAS application after validating:
+- That the specificied version of application is compatible to rollback from the current version. 
+- That the specificied version of application is compatible with the running MAS core platform.
+
+It will rollback the Manage Application to the desired version.
+It will validate that the Manage Application has been successfully reconciled at the rolled back version.
+
+Role Variables
+-------------------------------------------------------------------------------
+### mas_instance_id
+Set the instance ID for the MAS installation where you wish to rollback the application.
+
+- **Required**
+- Environment Variable: `MAS_INSTANCE_ID`
+- Default Value: None
+
+### rollback_mas_app
+When set to `true` will ensure that the role performs rollback operation.
+
+- optional
+- Environment Variable: `ROLLBACK_MAS_APP`
+- Default: True
+
+### verify_app_version
+When set to `true` will ensure that the role checks the current Manage Appliation version matches with specified version. 
+
+- optional
+- Environment Variable: `VERIFY_APP_VERSION`
+- Default: False
+
+### mas_app_version
+The version you wish to rollback to.  Built-in validation will ensure that the rollback will only proceed if a supportable rollback path is chosen. It is required when any of the ROLLBACK_MAS_APP and VERIFY_APP_VERSION variables is set to `true`.
+
+- **Required**
+- Environment Variable: `MAS_APP_VERSION`
+- Default Value: None
+
+### mas_app_id
+The name of the Maximo Application Suite Application. This will be used to lookup for application namespace and resources.
+
+- **Required**
+- Environment Variable: `MAS_APP_ID`
+- Default Value: manage
```

Comment:
the default isn't manage, the default is None. We should add to this description that only manage is currently supported.

Reply:
Agreed. In our pipeline we need to pass in Manage